be/btree.c

```diff
@@ -476,9 +489,11 @@ static void be_btree_split_child(struct m0_be_btree *btree,
 	i = 0;
 	while (i < new_child->bt_num_active_key) {
 		new_child->bt_kv_arr[i] = child->bt_kv_arr[i + BTREE_FAN_OUT];
+		new_child->bt_ik[i] = child->bt_ik[i + BTREE_FAN_OUT];
+		new_child->allocated[i] = child->allocated[i + BTREE_FAN_OUT];
```

Comment:
Same comment as above. This is because bt_ik and allocated DO NOT have a direct relation to bt_kv_arr elements.